test/saithrift/tests/sail3.py

```diff
@@ -2025,3 +2025,630 @@ def runTest(self):
             attr_value = sai_thrift_attribute_value_t(u16=1)
             attr = sai_thrift_attribute_t(id=SAI_PORT_ATTR_PORT_VLAN_ID, value=attr_value)
             self.client.sai_thrift_set_port_attribute(port1, attr)
+
+@group('l3')
+@group('ecmp')
+class L3IPv4EcmpGroupMemberTest(sai_base_test.ThriftInterfaceDataPlane):
+    def runTest(self):
+        '''
+        Description:
+        i) L3IPv4EcmpGroupMemberTest I:
+        Create a VRF with IPv4 and IPv6 enabled. Create three router interfaces in
+        the same VRF. Create a route (24 mask) with nhop and neighbor entry in two
```

Comment:
Here I think we need to rephrase. The RIFs are added to next-hops. Then we need to create a next-hop group and add next-hops to those groups, which

Reply:
rephrased as per comments